TheChocolateRiversDB/4 data.sql

```diff
@@ -0,0 +1,67 @@
+go
+use ChocolateRiverDB
+go
+delete Chocolate
+go
+
+insert Chocolate (ChocolateType, Flavor, Shape, RecipeSource, YearPutonMarket, ChocolateWeight, DateSold)
```

Comment:
-- LL once you change your expiration column, include the expiration in the insert